History/js/new.js

```diff
@@ -5,24 +5,6 @@
 /* global historyModal:true */
 
 /**
- * Message handlers for integration with extension framworks.
- */
-var messaging = {
-  
-  /**
-   * Attach the message listeners to the interface between the extension
-   * and the injectable application.
-   */
-  initialize: function() {
-    privlyExtension.initialContent = function(){};
-    privlyExtension.messageSecret = function(){};
-      
-    // Initialize message pathway to the extension.
-    privlyExtension.firePrivlyMessageSecretEvent();
```

Comment:
@SummerWish No more message secret ? I think it might be needed for FF since messages passed b/w content scripts and privly applications will happen via `parent.postMessage`